Combobox.js

```diff
@@ -1010,8 +1031,7 @@ define([
 					nd.id = "d-combobox-item-" + idCounter++;
 				}
 
-				var input = this._popupInput || this.inputNode;
-				input.setAttribute("aria-activedescendant", nd.id);
+				this.setAttribute("aria-activedescendant", nd.id);
```

Comment:
I'm confused about this change, from looking at https://www.w3.org/TR/wai-aria-1.1/#h-combobox it seems like `aria-activedescendant` should be on the `<input>`.